templates/layout-horizontal.html.twig

```diff
@@ -35,15 +35,9 @@ Enjoy your theme!
                     <span class="navbar-toggler-icon"></span>
                 </button>
                 {% block app_logo %}
-                    <h1 class="navbar-brand d-none-navbar-horizontal pe-0 pe-md-3{% block navbar_brand_class %} navbar-brand-autodark{% endblock %}">
-                        <a href="{{ path('tabler_welcome'|tabler_route) }}">
-                            {% if tabler_bundle.getLogoUrl() is not empty %}
-                                <img src="{% if '://' in tabler_bundle.getLogoUrl() %}{{ tabler_bundle.getLogoUrl() }}{% else %}{{ asset(tabler_bundle.getLogoUrl()) }}{% endif %}" width="110" height="32" alt="Logo" class="navbar-brand-image">
-                            {% else %}
-                                <i class="far fa-smile" style="font-size: 32px"></i>
-                            {% endif %}
-                        </a>
-                    </h1>
+                    {% embed '@Tabler/embeds/logo.html.twig' %}
```

Comment:
Have you tried if it is possible to override `navbar_brand_classes` in your project?  I assume not and this would be a BC break. 

Shall we probably only put the inner `{% if tabler_bundle.getLogoUrl() is not empty %}` into the embed and leave the `h1` in here?

Reply:
Ahhhh... Here we go again.. I forget that but we cannot override block of an embed..
Should I revert my last commit ?

Reply:
Either that or we move only this part to the embed, leaving the h1 in the original layouts
```
{% if tabler_bundle.getLogoUrl() is not empty %}
                                <img src="{% if '://' in tabler_bundle.getLogoUrl() %}{{ tabler_bundle.getLogoUrl() }}{% else %}{{ asset(tabler_bundle.getLogoUrl()) }}{% endif %}" width="110" height="32" alt="Logo" class="navbar-brand-image">
                            {% else %}
                                <i class="far fa-smile" style="font-size: 32px"></i>
                            {% endif %}
```

Reply:
👍